apps/files/src/components/FilesApp.vue

```diff
@@ -123,7 +143,6 @@ export default {
     FileActionsTab,
     FileList,
     FileUpload,
-    FileUploadProgress,
```

Comment:
Kill the component as well - please - THX

Reply:
I'll take care ....

Reply:
allready cleaned that mess